src/pybind/mgr/cephadm/agent.py

```diff
@@ -51,6 +53,30 @@ def __init__(self, mgr: "CephadmOrchestrator") -> None:
         self.server_addr = self.mgr.get_mgr_ip()
         super(CherryPyThread, self).__init__(target=self.run)
 
+    def configure_cherrypy(self) -> None:
+        cherrypy.config.update({
+            'environment': 'production',
+            'server.socket_host': self.server_addr,
+            'server.socket_port': self.server_port,
```

Comment:
I know it's not this PR's issue - but is the server port being addressed 7150 is too low on dense OSD nodes ?

Reply:
7150 is the port of the cephadm agent and right now is hardcoded. Since it's not related to this PR I think we can try to make it configurable in a separate PR. 

```
class CherryPyThread(threading.Thread):
    def __init__(self, mgr: "CephadmOrchestrator") -> None:
        self.mgr = mgr
        self.cherrypy_shutdown_event = threading.Event()
        self.ssl_certs = SSLCerts(self.mgr)
        self.server_port = 7150

```